app/controllers/api/v1x1/icons_controller.rb

```diff
@@ -0,0 +1,54 @@
+module Api
+  module V1x1
+    class IconsController < ApplicationController
```

Comment:
Should we just be inheriting from the V1 controller and then only overwrite the methods that actually changed? That's basically what we're doing in the v1x1.rb file except not actually changing any methods, right?

Reply:
It isn't in the `v1x1.rb` file so in this case it's a brand new instance of the controller only available for v1x1.

Reply:
Ok, but if you do `class IconsController < Api::V1::IconsController` then we don't have to redefine `#create` and `#destroy` and all the other methods that haven't changed, right?

Reply:
Yes that will work. But we would potentially run into management problems then if v1x0 was no longer supported and deleted. We would then have to remember to add those 'deleted' methods back into the v1x1 version of the controller.

This feels like the safer ( albeit more duplicated ) option.